vm-setup/roles/v1aX_integration_test/tasks/move.yml

```diff
@@ -82,6 +98,13 @@
   - name: Reinstate Ironic Configmap
     shell: "mv {{ BMOPATH }}/ironic-deployment/keepalived/ironic_bmo_configmap.env.orig {{ BMOPATH }}/ironic-deployment/keepalived/ironic_bmo_configmap.env"
 
+  - name: Label BMO CRDs in target cluster.
+    shell: 'kubectl --kubeconfig /tmp/kubeconfig-{{ CLUSTER_NAME }}.yaml label crds baremetalhosts.metal3.io {{ item }} --overwrite '
+    with_items:
+      - clusterctl.cluster.x-k8s.io=""
+      - cluster.x-k8s.io/provider="metal3"
+    when: CAPM3_VERSION == "v1alpha5"
+
```

Comment:
is this for pivoting back and same question with move label here too?

Reply:
Yes, CRD is labeled for pivoting back.

Reply:
then would it make sense to instead do it within move_back.yaml?

Reply:
Yes, I think we could do that in move_back.yaml. But in general label is set for all components that we want clusterctl to handle. 

```
// ClusterctlLabelName is applied to all components managed by clusterctl.
	ClusterctlLabelName = "clusterctl.cluster.x-k8s.io"
```

IMO setting these labels is part of the move and pivot operations. Labels should be set right after the move is done for clusterctl to be able to manage the component. So I would leave it here!